tests/trainer/logging_/test_train_loop_logging.py

```diff
@@ -715,19 +715,15 @@ def on_validation_epoch_end(self):
             assert all(v == 3 for v in self.trainer.callback_metrics.values())
 
         def on_train_batch_start(self, batch, batch_idx):
-            assert self.trainer._results.batch_size == 2
```

Comment:
Why did you remove this ?

Reply:
because reduce_fx = 'sum', so we don't track the batch_size in such a case.